### PR TITLE
Fix columns, typo

### PR DESCRIPTION
**Pull request by KaminKevCrew**

### CODE REVIEW

File: server/models/campaigns.py [hunk 1]
```diff
@@ -10,8 +10,10 @@ class CampaignModel(db.Model):
     id = db.Column(db.BigInteger, primary_key=True)
     # Figure out how to make a unique clause where the combination between user_id and campaign title must be unique
     title = db.Column(db.String(200), unique=True, nullable=False)
-    # steps = db.relationship("StepModel", backref="campaign", lazy=True)
+    steps = db.relationship("StepModel", backref="campaign", lazy=True)
     user_id = db.Column(db.BigInteger, db.ForeignKey("users.id"), nullable=False)
+    reply_count = db.Column(db.BigInteger, nullable=False, default=0)
```
Comment (kimcarey) [inline server/models/campaigns.py:16]: I think what we need for reply_count and prospect_count to be calculated based on what's in the prospects_step_join table at any given moment.. essentially a group_by and count per this doc.. https://stackoverflow.com/questions/1052148/group-by-count-function-in-sqlalchemy .. I'm not sure exactly how sqlAlchemy does this, but I recall this from learning SQL.

**Comment by kimcarey** [comment]
Couple more issues from the PR already merged:

1) `campaign_resources.py` is created, but isn't added to the app.py list of resources. I already started a Campaigns(Resource) setup per our chat earlier and started implementing the endpoints. Maybe you committed it by mistake? I don't have the endpoint yet to edit a campaign in my merged code, so you could just use your code for that and move it into my implementation in `resources.py`?

2) `campaign_resources.py` has an `import login_required from Resource` at the top, but that's an error. It would be `import login_required from resources`

File: client/src/App.js [hunk 2]
```diff
@@ -41,6 +41,7 @@ function App() {
       <BrowserRouter>
         {user !== null ? <LoggedInNavBar /> : ""}
         <UserContext.Provider value={{ user, setUser }}>
+          <Route path="/camp_show" component={CampaignShow}/>
```
Comment (CharlieLe3) [inline client/src/App.js]: guessing this is just to test it?

File: server/models/prospects.py [hunk 3]
```diff
@@ -13,7 +13,8 @@ class ProspectModel(db.Model):
     firstName = db.Column(db.String(120), nullable=True)
     lastName = db.Column(db.String(120), nullable=True)
     userId = db.Column(db.BigInteger, db.ForeignKey("users.id"), nullable=False)
-    # prospect_step_join = db.relationship('ProspectStepModel', backref='prospect_step', lazy=True)
+    prospect_step_join = db.relationship('ProspectStepModel', backref='prospect_step', lazy=True)
+    prospect_step_join = db.relationship('ProspectCampaignModel', backref='prospect_campaign', lazy=True)
```
Comment (CharlieLe3) [inline server/models/prospects.py]: Think maybe u meant to do prospect_campaign_join?

File: server/campaign_step_resources.py [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+                'status': 'fail',
+                'message': 'Something went wrong. Please try again.'
+            }
+            return responseObject, 500
+
+class StepUpdate(Resource):
+    def post(self):
+        data = request.json
+        step = StepModel.find_by_campaign(data['campaign_id'])
+        StepModel.update_step(step)
```
Comment (CharlieLe3) [inline server/campaign_step_resources.py]: Doesn't this just update the step to be the same as it was before? i could be thinking about this wrong though

File: client/src/App.js (hunk 2 above)
Comment (KaminKevCrew) [reply client/src/App.js]: Yeah it was - fixed.

File: server/models/prospects.py (hunk 3 above)
Comment (KaminKevCrew) [reply server/models/prospects.py]: Indeed - fixed

File: server/campaign_step_resources.py (hunk 4 above)
Comment (KaminKevCrew) [reply server/campaign_step_resources.py]: You're 100% right; fixed now.

File: server/prospect_campaign_join_resources.py [hunk 5]
```diff
@@ -0,0 +1,249 @@
+from typing import List, Any
```
Comment (CharlieLe3) [inline server/prospect_campaign_join_resources.py]: I think you copy and pasted all of Users in here, but isn't this supposed to be prospect campaign?

File: server/prospect_step_join_resources.py [hunk 6]
```diff
@@ -0,0 +1,278 @@
+from typing import List, Any
```
Comment (CharlieLe3) [inline server/prospect_step_join_resources.py]: Same here

File: server/prospect_campaign_join_resources.py (hunk 5 above)
Comment (KaminKevCrew) [reply server/prospect_campaign_join_resources.py]: Those were supposed to be in a different branch. Oops! Fixed now.

**Review by CharlieLe3 — APPROVED** [review]

**Pull request merged**